wp/lib/bap_wp/src/constraint.mli

```diff
@@ -101,7 +101,7 @@ val mk_clause : t list -> t list -> t
 
 (** [eval c ctx] evaluates the constraint [c] to a Z3 expression,
     using the standard Z3 encoding of if-then-else and clauses. *)
-val eval : t -> Z3.context -> z3_expr
+val eval : ?debug:(string list) -> t -> Z3.context -> z3_expr
```

Comment:
Are the available debug options `"solver", "global", "constr.t", "eval"`? Can we add a quick comment about that here?

Reply:
Those are the four options.
eval can use the "eval" option, if set, for printing certain statistics.
I'll add that to the comments.

Reply:
Thanks!